### ISSUE TITLE
Locust not reading from locust.conf

### ISSUE THREAD

**Issue by jreich218** [locustio/locust#1813]
### Describe the bug

Locust not reading from .locust.conf
### Expected behavior

The "All available configuration options" table of the docs seem to make it clear that LOCUST_HOST and host are on the same footing. The former is for .env and the latter (according to the docs just above said table) can go in locust.conf
### Actual behavior

`host` never has in it the base URI I assign in locust.conf
### Steps to reproduce
 
Any basics Locust API test where you place the base URI of the target machine in the host variable in locust.conf and expect it to work just as if you had placed it in a .env file into the environment variable LOCUST_HOST
### Environment

- OS:
- Python version:
- Locust version:
- Locust command line that you ran:
- Locust file contents (anonymized if necessary):

**Comment by cyberw** [comment]
Locust reads from config files (./locust.conf and ~/.locust.conf) just fine for me, but maybe you have run into some edge case?

**Comment by jreich218** [comment]
@cyberw Well, when I saw this was from the guy from the video in the docs I knew I was definitely doing something wrong immediately :)

So basically I had left this

```
    env_file:
      - ./.env
```

in my docker-compose.yml file even after I switched to locust.conf. Needless to say, when I switched `.env` to `locust.conf` things got better.

Interestingly, I noticed that when I use LOCUST_HOST (and this is still about locust.conf), then the Flask interface populates the host URI automatically and, of course, the endpoints get that same URI for their base during testing.

But if I use `host` in the locust.conf file... then the Flask interface has an empty field for the host URI (but after I paste it in to the browser, the endpoints are definitely properly wired up under the hood and everything does work perfectly).

Clearly there's no bug to speak of. And I'm assuming all of the above is a feature... it all works for me...

Warm regards...
